api-reference/v2/api-reference-guide.md

```diff
@@ -0,0 +1,213 @@
+# API reference guide
+
+This `api-reference-guide` document provides extra writing tips describing **HOW** to fill in each of the sections within the [`api-reference-template`](api-reference-template.md) file.
+
+- _Document version_: 0.0.1
+- _Last updated_: June 27, 2022
+
+## Overview
+
+API references are technical manuals that provide the specifications and integration instructions of APIs. An accurate, concise, well-structured API documentation facilitates the adoption of APIs and increases the overall user experience.
+
+The [`api-reference-template`](api-reference-template.md) is designed to help you build the API documentation efficiently and keep it consistent in both format and appearance.
+
+## Before writing the API references
+
+- Familiarize yourself with the API protocol. If possible, try making some API calls in a testing environment. This API reference template works best with the REST API protocol. If you are working with other protocols, you may need to adjust the organization of some sections.
+- Interact with your API developers to learn about the APIs that you are documenting. Understand the data models and the logical connections between the API endpoints.
+- Discuss the logic of grouping the API endpoints. Although grouping the endpoints by resource type is a common practice, as used in this template, it is also possible to group them by use case or other characteristics that better suit the user's needs.
+- Conduct some user research about how the audience of your documentation would use the APIs. Identify the programming languages that your audience would most likely adopt to interact with your APIs.
+- Explore the possibility of auto-generating the API references.
+- Optimize the visual presentation of the API references by applying customized stylesheet, such as using syntax highlighting, table of contents, multi-column layout, etc.
+
+## Document structure
+
+Assuming that your API documentation set includes references of many API endpoints that are organized into groups, this template includes three parts:
+
+- The [API reference overview section](#about-the-api-reference-overview-section) provides reference information for the whole API documentation set. It describes product-wise API specifications, such as protocol, authorization method, versioning, status code, etc.
```

Comment:
nit: You could remove `etc.`
I think you are describing the common fields in the API specification which apply to the entire API:

The Overview section in the API reference describes the API components [fields] which apply to the entire API [or all endpoints in the API].